test/e2e/common/pods.go

```diff
@@ -600,7 +610,12 @@ var _ = framework.KubeDescribe("Pods", func() {
 		}
 	})
 
-	It("should have their auto-restart back-off timer reset on image update [Slow][NodeConformance]", func() {
+	/*
+	  Release : v1.12
+	  Testname: Pods: back-off-timer reset.
+	  Description: Updating the image in a container must reset the back-off-timer in the pod.
+	*/
+	framework.ConformanceIt("should have their auto-restart back-off timer reset on image update [Slow][NodeConformance]", func() {
```

Comment:
[This averages 7min per run](https://k8s-testgrid.appspot.com/sig-release-master-blocking#gci-gke-slow&graph-metrics=test-duration-minutes&include-filter-by-regex=auto-restart&width=5)